tests/env/Dockerfile-beanstalkd.yaml

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:rolling
+FROM ubuntu:20.04
```

Comment:
Do not change the Dockerfile ubuntu version.

Reply:
it doesnt even build with ubuntu:rolling

Reply:
It's your system issue. All actions work correctly.
![image](https://user-images.githubusercontent.com/8040338/143037975-59ef3eb4-65d6-4959-977f-1e2649338e63.png)

Reply:
docker dosn't work this way
![image](https://user-images.githubusercontent.com/1983796/143039444-9b7d1bd1-7890-4f45-a8d4-db4567ca5651.png)

Reply:
Existing dockerfile doesn't work for the latest ubuntu:rolling image from dockerhub

Reply:
It's your OS issue. Everything works correctly on my system as well as in the GitHub Actions.

Reply:
Just prune the existing cached image and fetch the latest one. It wont work. Just because docker run the commands inside the container, neither on the host os.

Reply:
1. I nuked everything with the `docker system prune -a`, screenshot made after that.
2. Everything works correctly on every GA run after every single commit. All docker images were built from scratch in the GA. It's completely your OS/Docker issue.